test/tftp/tftp_test.rb

```diff
@@ -5,7 +5,8 @@
 class TftpTest < Test::Unit::TestCase
   def setup
     @tftp = Proxy::TFTP::Server.new
-    Proxy::TFTP::Plugin.load_test_settings(:tftproot => "/some/root")
+    Proxy::TFTP::Plugin.load_test_settings(:tftproot => "/some/root",
+                                           :tftp_image_path => "/another/root")
```

Comment:
I don't see this setting anywhere else. Is this a leftover or something you intend to implement?

Reply:
You are right. Removed it.